README.md

```diff
@@ -32,6 +31,17 @@ Rickshaw needs the following:
     - RS_EMAIL Environment variable or --email option with user's email address "my-email@email.domain"
     - RS_TAGS Environment variable or --tags containing a comma-separated list of words that are relevant to the run
     - RS_DESC Environment variable or --desc containing a free form description of the purpose, conditions, or any other relevant information about this test.
+
+Other, optional paramers include:
```

Comment:
What about the mode of running different iterations and/or samples on different nodes?

Reply:
Are you referring to the parallel execution idea?  I have definitely not put much thought into that yet.  If we did support that, I wonder if it should be a different endpoint type, like a special version of "ssh" endpoint.  I don't know...  Not a real concern at this point.

Reply:
I think so.  I guess I thought it would be similar in concept to your current idea of i-s vs. s-i.  I would have thought it would be endpoint agnostic, but maybe I don't fully grasp the implications.